Codegen/test/TestTypeCobolDomainCodegen.cs

```diff
@@ -0,0 +1,1034 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.Collections.Generic;
+using System.IO;
+using TypeCobol.Codegen.Config;
+using TypeCobol.Codegen.Skeletons;
+using TypeCobol.Compiler; // DocumentFormat
+using TypeCobol.Compiler.Domain;
+using TypeCobol.Compiler.Parser;
+using TypeCobol.Tools; // CodeElementDiagnostics
+using TypeCobol.Tools.Options_Config;
+
+namespace TypeCobol.Codegen.Domain
+{
+    [TestClass]
+    public class TestTypeCobolDomainCodegen
+    {
+        public static bool UseSkeleton = false;
+        public static TypeCobolConfiguration DefaultConfig = null;
+        public static ProgramSymbolTableBuilder Builder = null;
+        public static NodeListenerFactory BuilderNodeListenerFactory = null;
+        public static string DefaultIntrinsicPath = null;//@"C:\TypeCobol\Sources\##Latest_Release##\Intrinsic\Intrinsic.txt";
+
+        [TestInitialize]
+        public void TestInitialize()
+        {
+            SymbolTableBuilder.Root = null;
+            //Create a default configurations for options
+            DefaultConfig = new TypeCobolConfiguration();
+            if (File.Exists(DefaultIntrinsicPath))
+            {
+                DefaultConfig.Copies.Add(DefaultIntrinsicPath);
```

Comment:
DefaultIntrinsicPath should be a path to a single TypeCobol file.
Why add this to the list of COPY folders ?

Reply:
I can remove that because at the beginning I used to load Intrinsics as COPY I think, but sure it is a non sens.